tikv-configuration-file.md

```diff
@@ -1318,14 +1318,15 @@ Configuration items related to `raftdb`
 
 Configuration items related to Raft Engine.
 
-> **Warning:**
->
-> Raft Engine is an experimental feature. It is not recommended to use it in the production environment.
+> **Note:**
+> 
+> - When you enable Raft Engine first time, you need to wait tens of seconds, because TiKV switches to the data format of Raft Engine from the one of RocksDB.
```

Comment:
```suggestion
> - When you enable Raft Engine for the first time, you need to wait tens of seconds because TiKV switches to the data format of Raft Engine from the one of RocksDB.
```